packages/pygments/meta.yaml

```diff
@@ -0,0 +1,9 @@
+package:
+  name: pygments
```

Comment:
```suggestion
  name: Pygments
```

This needs to match the tarball name.  Also, the `packages/pygments` directory needs to move to `packages/Pygments`.

Reply:
Updated. Thanks!